rasa/nlu/tokenizers/jieba_tokenizer.py

```diff
@@ -102,14 +134,9 @@ def copy_files_dir_to_dir(input_dir: Text, output_dir: Text) -> None:
         for target_file in target_file_list:
             shutil.copy2(target_file, output_dir)
 
-    def persist(self, file_name: Text, model_dir: Text) -> Optional[Dict[Text, Any]]:
-        """Persist this model into the passed directory."""
-
-        # copy custom dictionaries to model dir, if any
-        if self.dictionary_path is not None:
-            target_dictionary_path = os.path.join(model_dir, file_name)
-            self.copy_files_dir_to_dir(self.dictionary_path, target_dictionary_path)
-
-            return {"dictionary_path": file_name}
-        else:
-            return {"dictionary_path": None}
+    def persist(self) -> None:
```

Comment:
don't we need to call this as part of `train`?

Reply:
didn't we cover this here? https://github.com/RasaHQ/rasa/pull/9562#discussion_r703605128 😁 
(or am I missing something?)

Reply:
Wait - but something has to call `persist` then? I thought we'd add a `train` to the tokenizer where call `persist`

Reply:
Oh you mean a `train` method which just calls `persist`. I was assuming we would call `persist` directly as it wont be called in the same way anyway. i.e. wont have any inputs or outputs.

Reply:
Ooooh, yes,we can also do that. For consistency (and less edge conditions in the recipe) a `train` would be nicer but whatever 👍🏻